x-pack/filebeat/module/suricata/eve/test/eve-alerts.log-expected.json

```diff
@@ -0,0 +1,1250 @@
+[
+    {
+        "@timestamp": "2018-10-03T14:42:44.836Z", 
+        "destination.geo.city_name": "Norwell", 
+        "destination.geo.continent_name": "North America", 
+        "destination.geo.country_iso_code": "US", 
+        "destination.geo.location.lat": 42.1508, 
+        "destination.geo.location.lon": -70.8228, 
+        "destination.geo.region_iso_code": "US-MA", 
+        "destination.geo.region_name": "Massachusetts", 
+        "destination.ip": "93.184.216.34", 
+        "destination.port": 80, 
+        "event.type": "alert", 
+        "fileset.module": "suricata", 
+        "fileset.name": "eve", 
+        "http.request.method": "GET", 
+        "http.response.status_code": "200", 
+        "input.type": "log", 
+        "offset": 0, 
+        "prospector.type": "log", 
+        "source_ecs.ip": "192.168.1.146", 
+        "source_ecs.port": 32858, 
+        "suricata.eve.alert.action": "allowed", 
+        "suricata.eve.alert.category": "Attempted Information Leak", 
+        "suricata.eve.alert.gid": 1, 
+        "suricata.eve.alert.rev": 4, 
+        "suricata.eve.alert.severity": 2, 
+        "suricata.eve.alert.signature": "ET POLICY curl User-Agent Outbound", 
+        "suricata.eve.alert.signature_id": 2013028, 
+        "suricata.eve.app_proto": "http", 
+        "suricata.eve.dest_ip": "93.184.216.34", 
+        "suricata.eve.dest_port": 80, 
+        "suricata.eve.event_type": "alert", 
+        "suricata.eve.flow.bytes_toclient": 1654, 
+        "suricata.eve.flow.bytes_toserver": 347, 
+        "suricata.eve.flow.pkts_toclient": 3, 
+        "suricata.eve.flow.pkts_toserver": 4, 
+        "suricata.eve.flow.start": "2018-10-03T14:42:44.613469+0000", 
+        "suricata.eve.flow_id": 2191386088856669, 
+        "suricata.eve.http.hostname": "example.net", 
+        "suricata.eve.http.http_content_type": "text/html", 
+        "suricata.eve.http.http_method": "GET", 
+        "suricata.eve.http.http_user_agent": "curl/7.58.0", 
+        "suricata.eve.http.length": 1121, 
+        "suricata.eve.http.protocol": "HTTP/1.1", 
+        "suricata.eve.http.status": 200, 
+        "suricata.eve.http.url": "/", 
+        "suricata.eve.in_iface": "enp0s3", 
+        "suricata.eve.proto": "TCP", 
+        "suricata.eve.src_ip": "192.168.1.146", 
+        "suricata.eve.src_port": 32858, 
+        "suricata.eve.timestamp": "2018-10-03T14:42:44.836744+0000", 
+        "suricata.eve.tx_id": 0, 
+        "tags": [
+            "suricata"
+        ], 
+        "url.host.name": "example.net", 
```

Comment:
@andrewkroh I see this is still `url.host.name`? Is it planned to change it to `url.hostname`?

Reply:
There's a test failing on this field so I better fix it. 👷  

> Key 'url.host.name' found in event is not documented!